packages/cli/lib/prompts/selectPublicAppPrompt.js

```diff
@@ -20,7 +24,9 @@ const fetchPublicAppOptions = async (accountId, accountName, migrateApp) => {
     if (
       !filteredPublicApps.length ||
       (migrateApp &&
-        !filteredPublicApps.find(app => !app.preventProjectMigrations))
+        !filteredPublicApps.find(
```

Comment:
What exactly is this chunk of logic doing? It it confirming that there are valid items in the list? If so we may want to use `array.some()` here

Reply:
Yes, it's confirming there are valid items in the list, in response to your feedback that we shouldn't display a dropdown menu where everything is disabled. I can change to `some`.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1114/commits/575fa8706bfc21cc7c13a98b76aee15e4fb411f3